facebook/facebook-instant-articles-rules-editor
# Add section for transformation settings

This PR adds a new section to define additional settings for the transformation.

Here is a summary of the changes:
- Use an accordion to separate the existing rules configuration from the new settings
- Add a new accordion pane to define:
  - Style name
  - Audience Network Placement ID
  - Ads raw HTML
  - FB Pixel ID
  - Analytics raw HTML
 - Upgrade the minimum SDK version used for the preview, so that the new fields are processed and included in the generated IA markup
 - Add unit tests

## Testing Plan

-  Upgrade the IA SDK version in the webserver folder by running `composer install`
- Load a sample article
 - Fill out the required selectors
 - Switch to the new settings section and specify
   - Style Name
     - Verify the new name appears in the generated markup
   - FB Pixel ID
     - Verify the ID appears in the generated markup
   - Audience Network Placement ID
     - Verify the ID appears in the generated markup
 - Clear the FB Pixel ID and the Audience Network Placement ID fields
 - Set values for both of the raw HTML fields
   - Verify the values are in the generated markup
